reprospect/test/sass/matchers/move32.py

```diff
@@ -0,0 +1,60 @@
+import sys
+import typing
+
+from reprospect.test.sass.composite import Fluentizer, instruction_is
+from reprospect.test.sass.instruction import (
+    InstructionMatch,
+    InstructionMatcher,
+    OpcodeModsWithOperandsMatcher,
+    PatternBuilder,
+)
+from reprospect.test.sass.instruction.move import MovMatcher
+from reprospect.tools.sass.decode import Instruction
+
+if sys.version_info >= (3, 12):
+    from typing import override
+else:
+    from typing_extensions import override
+
+
+class Move32Matcher(InstructionMatcher):
```

Comment:
We could also parameterise it on the size. This could also be left for later.

Reply:
Sure for later.